### PR TITLE
Draft: Fixes for using postgresql database

### PR DESCRIPTION
**Pull request by rhabacker**
The commit in this pull request fixes issues mentioned at https://github.com/aimeos/aimeos-typo3/issues/200

After the issue mentioned at https://github.com/aimeos/aimeos-typo3/issues/200#issuecomment-1466887825 is fixed I will remove the draft state of this pull request.

### CODE REVIEW

File: Classes/Setup.php [hunk 1]
```diff
@@ -170,18 +170,27 @@ public static function schema(array $sql) : array
 
         foreach ($connectionNames as $connectionName) {
             $conn = $ctx->db($connectionName);
+            $dbtype = $conn->getRawObject()->getParams()['adapter'];
```
Comment (aimeos) [inline Classes/Setup.php:173]: If you use `$ctx->config('resource/' . $connectionName . '/adapter')`, you don't have to distinguish between `pgsql` and `pdo_pgsql` any more

Comment (rhabacker) [reply Classes/Setup.php]: Fixed, thanks for this pointer.

File: Resources/Private/Config/resource.php [hunk 2]
```diff
@@ -2,17 +2,32 @@
 
 $publicPath = \TYPO3\CMS\Core\Core\Environment::getPublicPath();
 $defaultConnection = $GLOBALS['TYPO3_CONF_VARS']['DB']['Connections']['Default'];
+$driver = $defaultConnection['driver'];
+
+switch($driver) {
+	case 'pdo_mysql': $adapter = 'mysql'; break;
```
Comment (aimeos) [inline Resources/Private/Config/resource.php:8]: `mysqli` as option could be necessary too

**Comment by aimeos** [comment]
Perfect and thank you for your PR! :-)

**Pull request merged**